app/server/models/User.js

```diff
@@ -72,7 +72,7 @@ var confirmation = {
     }
   },
   wantsHardware: Boolean,
-  hardware: String,
+  hardwareRequest: String,
```

Comment:
Once again, maybe don't change this.... especially if you're making another field called hardware.